tests/phpunit/tests/theme/wpThemeJsonResolver.php

```diff
@@ -365,12 +369,56 @@ function test_get_user_data_from_wp_global_styles_does_not_use_uncached_queries(
 		$this->assertNotEmpty( $user_cpt );
 
 		$query_count = count( $this->queries );
-		for ( $i = 0; $i < 3; $i++ ) {
-			WP_Theme_JSON_Resolver::get_user_data_from_wp_global_styles( wp_get_theme() );
+		for ( $i = 0; $i < 3; $i ++ ) {
+			$new_user_cpt = WP_Theme_JSON_Resolver::get_user_data_from_wp_global_styles( wp_get_theme() );
 			WP_Theme_JSON_Resolver::clean_cached_data();
+			$this->assertSameSets( $user_cpt, $new_user_cpt );
 		}
 		$query_count = count( $this->queries ) - $query_count;
 		$this->assertEquals( 0, $query_count, 'Unexpected SQL queries detected for the wp_global_style post type' );
 		remove_filter( 'query', array( $this, 'filter_db_query' ) );
 	}
+
+	/**
+	 * @covers WP_Theme_JSON_Resolver::get_user_data_from_wp_global_styles
+	 * @ticket 55392
```

Comment:
Per [56070](https://core.trac.wordpress.org/ticket/56070), @ticket should precede @covers and there should be a line break between them.

i.e.
```suggestion
	 * @ticket 55392
	 *
	 * @covers WP_Theme_JSON_Resolver::get_user_data_from_wp_global_styles
```

This also applies to the other test method.